new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -613,6 +614,98 @@ public void testDomainIntrospectVersionRolling() {
 
   }
 
+  /**
```

Comment:
May be modify the description step-by-step 
a. Create new WebLogic Cluster using WLST 
b. Patch the Domain Resource 
c. Update the Introspector version
d. Make sure the Servers in the new WebLogic cluster comes up with out any affect to  running  servers on pre-existing WebLogic cluster

Reply:
Good writeup. Modified javadoc.